src/v/redpanda/admin_server.cc

```diff
@@ -4101,6 +4102,69 @@ admin_server::get_partition_cloud_storage_status(
     co_return map_status_to_json(*status);
 }
 
+ss::future<std::unique_ptr<ss::httpd::reply>> admin_server::dump_manifest(
+  std::unique_ptr<ss::httpd::request> req,
+  std::unique_ptr<ss::httpd::reply> rep) {
```

Comment:
this will break in upstream/dev due to changes in seastar: needs to be `ss::http::` scoped instead of `httpd`.

Reply:
Thanks for the call-out. Will update.